tools/build-tools/src/bumpVersion/context.ts

```diff
@@ -98,6 +98,7 @@ export class Context {
         }
 
         const publishedPackageDependenciesPromises: Promise<void>[] = [];
+        // eslint-disable-next-line no-constant-condition
         while (true) {
```

Comment:
For a real fix, I think you could do something like
```ts
let pkg;
while ((pkg = pendingDepCheck.pop()) !== undefined) {
// ...
```

Reply:
Same for a few other places I see - though some might take more thought than is worth right now

Reply:
Yeah, I'm not trying to clean anything up right now -- just trying to enable the infrastructure so linting is possible. Follow up PRs can do more involved code changes.